src/Views/DriversView.vala

```diff
@@ -0,0 +1,114 @@
+/*-
+ * Copyright 2023 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ public class DriversView : AbstractInstallerView {
+    public signal void next_step ();
+
+    construct {
+        var image = new Gtk.Image.from_icon_name ("application-x-executable", Gtk.IconSize.DIALOG) {
+            pixel_size = 128,
+            valign = Gtk.Align.END
+        };
+
+        var title_label = new Gtk.Label (_("Install Drivers")) {
+            valign = Gtk.Align.START
+        };
+        title_label.get_style_context ().add_class (Granite.STYLE_CLASS_H2_LABEL);
+
+        var description_label = new Gtk.Label (_("Proprietary drivers are required for some devices, such as Wi-Fi adapters or graphics cards, to work properly.")) {
```

Comment:
Maybe a little more specific so folks know who this is and isn't for

```suggestion
        var description_label = new Gtk.Label (_("Some devices, such as NVIDIA® graphics or Broadcom® Wi-Fi adapters, may not function properly without proprietary drivers. Most devices do not require additional drivers.")) {
```

Reply:
> Most devices do not require additional drivers

A may be wrong, but Nvidia GPUs are more common than Amd or Intel and therefore most devices actually require these drivers.

Reply:
Ah by "devices" here I mean individual hardware devices, not computers. But I just poked around and [according to this random and not well researched source](https://www.hardwaretimes.com/amds-dgpu-market-share-4-1-nvidia-20-intels-igpu-share-at-62-q3-2021-overall-gpu-market-share/) integrated graphics is almost 80% of the market

Reply:
Oh makes sense then

Btw thanks for taking over the design of this, I couldn't have done it properly :)